### ISSUE TITLE
Align the API of GraphConvPredictor in the Tox21 example to that of sklearn.

### ISSUE THREAD

**Issue by delta2323** [pfnet-research/chainer-chemistry#49]
Currently, `GraphConvPredictor.predict` outputs an array of probabilities of being positive. But estimators in scikit-learn output labels use `predict_proba` for probabilities. In order for scikit-learn users to understand intuitively, we should align API to scikit-learn either by changing the API name to `predict_proba` or changing returned value to labels.

**Comment by corochann** [comment]
It has done by `Classifier` class introduced in v0.3.0. Can we close this issue?

**Comment by delta2323** [comment]
Yes we can
